blogposts/2023-02-10-adding-snapshot-testing-to-rnw-apps.md

```diff
@@ -0,0 +1,93 @@
+---
+post_title: "Adding Jest Snapshot Testing to React Native Windows Apps"
+author1: chiaramooney
+post_slug: 2023-02-15-adding-snapshot-testing-to-rnw-apps
+post_date: 2023-02-15 12:00:00
+categories: react-native
+tags: jest, react-native, testing, react-native-windows, tutorial
+summary: I added Jest Snapshot testing to a React Native Windows app with community modules. Here's a guide on how I did it!
+---
+
+[React Native Gallery](https://github.com/microsoft/react-native-gallery) is a React Native Windows app which showcases core and community module component samples. One function of the app is to validate whether a release has caused UI changes to our component set. In the past, this process was done manually by upgrading the app and walking through all sample pages to confirm that components were being rendered as expected, following the version upgrade. This process was time consuming and lacked accuracy. To automate more of this process, we've begun to add UI testing to React Native Gallery. In this blogpost, I'll walk through how I added Jest Snapshot testing to React Native Gallery. 
```

Comment:
Consider adding something in the intro that talks about how this guide is specifically about RNW (with a link to the RNW docs), but that this general flow is applicable to x-plat apps too. Additionally, the conclusion talks about RNW but the idea should also be introduced in the intro.